tests/gui/boards.feature

```diff
@@ -2,7 +2,7 @@ Feature: Trello UI Boards
     As a User I want to be able to make CRUD and other operations with boards
 
     Background: The user is logged in
-        Given the user goes to "/login"
+        Given the user goes to "login" page
```

Comment:
This should be made only in one step, "Given the user is logged in" and the navigation to the /login path should be made on the step implementation